data/json/items/armor/helmets.json

```diff
@@ -291,9 +296,11 @@
     "symbol": "[",
     "looks_like": "helmet_motor",
     "color": "yellow",
-    "covers": [ "HEAD" ],
-    "coverage": 85,
-    "encumbrance": 50,
+    "armor_portion_data": [
+      { "covers": [ "head" ], "coverage": 95, "encumbrance": 50 },
+      { "covers": [ "eyes" ], "coverage": 50, "encumbrance": 0 },
```

Comment:
I think this could probably use some eye encumbrance, you don't have quite as full of a field of view in this helmet.